### PR TITLE
First step for AutoUpdate

### PR DESCRIPTION
**Pull request by modelrailroader**
This creates a file named 'version_changes.php' in the setup-folder and replaces the old commands from update.php with requiring the new file. 

Otherwise, every database change will be added in two files soon, the planned autoUpdate-file and the manual update.php. 

It builds the first step for implementing an automatical update script #2492

### CODE REVIEW

**Comment by modelrailroader** [comment]
There is an unidentified problem with your CodeQL/Analyze-Check

File: phpmyfaq/setup/version_changes.php
Comment (thorsten) [inline phpmyfaq/setup/version_changes.php:1]: I would propose to rename the file to "migrations.php"

**Comment by thorsten** [comment]
> There is an unidentified problem with your CodeQL/Analyze-Check

I re-started the build and it worked.

File: phpmyfaq/setup/version_changes.php
Comment (modelrailroader) [reply phpmyfaq/setup/version_changes.php:1]: That sounds better, you’re right. Thanks!

**Comment by modelrailroader** [comment]
I renamed the file.

**Pull request merged**